### PR TITLE
CRUD API

### PR DESCRIPTION
**Pull request by priscilagonzalez**
- I also changed the name of the bundle and updated README.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -22,18 +22,113 @@ app.get('/:id', (req, res) => {
...
+    numberOfReviews,
+    duration,
+    groupSize,
+    includes: req.body.includes,
+    cuisine: req.body.cuisine,
+    hostedLanguages: req.body.hostedLanguages,
+    costPerPerson,
+    imageUrls: req.body.imageUrls,
+    videoUrl: req.body.videoUrl,
+  };
```
Comment (Kraig5) [inline server/index.js]: Maybe this code could be factored out and used for both put and post?

Comment (priscilagonzalez) [reply server/index.js]: I agree. Thanks!

**Comment by Kraig5** [comment]
lgtm

**Comment by azdoolittle** [comment]
lgtm

**Pull request merged**